Packs/PassiveTotal/Integrations/integration-PassiveTotal.yml

```diff
@@ -747,4 +771,5 @@ script:
     description: Search for SSL certificates based on field values
   runonce: false
 tests:
-  - no test
\ No newline at end of file
+- no test
+fromversion: 4.1.0
```

Comment:
Please add `deprecated: true` to the yml root

Reply:
done